packages/tupaia-web/src/views/ProjectsModal.tsx

```diff
@@ -29,19 +28,22 @@ const Wrapper = styled.div`
   display: flex;
   flex-direction: column;
   padding: 0.2rem 0 0;
-  width: 54rem;
+  width: 65rem;
   max-width: 100%;
+  text-align: left;
 `;
 
 const TagLine = styled.p`
-  margin: 0.625rem 0 2.5rem;
+  margin: 0.5rem 0.4rem 1.5rem;
+  width: 40%;
```

Comment:
I think it would be better to do `max-width: 26rem` because we don't want the TagLine to be 40% on mobile width screens.